libmscore/score.cpp

```diff
@@ -4589,7 +4589,22 @@ void MasterScore::updateExpressive(MasterSynthesizer* m, bool expressive, bool f
                   }
             }
 
-      // Rebuild midi mappings to be safe
+      }
+
+//---------------------------------------------------------
+//   rebuildAndUpdateExpressive
+//    implicitly rebuild midi mappings as well. Should be preferred over
+//    just updateExpressive, in most cases.
```

Comment:
By the way, if it should be preferred over the plain `updateExpressive`, does it make sense to make `updateExpressive` private to have more enforcement on this rule?

Reply:
No, because there are places where we need to rebuild, do some things, and _then_ update the expressive patches. So, we use the updateExpressive call instead there.

Reply:
Well, OK, then let's leave it as is.